frontend/src/config/screenConfig.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export default class ScreenConfig {
+  static height: number = 960;
+  static width: number = 804;
```

Comment:
マップの大きさを 15 x 13 に変更したので、ゲーム画面の大きさは以下の幅・高さになると思います！

```suggestion
  static width: number = 960;  // 15 x 64 = 960
  static height: number = 892;  // 13 x 64 + 60 = 892
```